core/src/main/java/com/yandex/yoctodb/util/UnsignedByteArrays.java

```diff
@@ -82,6 +82,12 @@ public static UnsignedByteArray from(final int i) {
         return from(Ints.toByteArray(i ^ Integer.MIN_VALUE));
     }
 
+    @NotNull
+    public static UnsignedByteArray from(final char c) {
+        final int ci = c ^ Character.MIN_VALUE;
```

Comment:
Redundant XOR with `Character.MIN_VALUE`

Reply:
Thank you, agree. I'll fix this.